db/version_set.cc

```diff
@@ -2221,6 +2223,7 @@ void VersionStorageInfo::EstimateCompactionBytesNeeded(
       assert(bytes_next_level == 0);
       if (level + 1 < num_levels_) {
         for (auto* f : files_[level + 1]) {
+          if (f->ingested_file) continue;
```

Comment:
I'm feeling should probably count ingested file for the next level, just not the current level? i.e.
```
// both level_size and next_level_size include ingested files
bytes_compact_to_next_level = (level_size - level_ingested_file) - level_target_size
estimated_compaction_needed_bytes_ += bytes_compact_to_next_level * (next_level_size/ level_size + 1)
```

Reply:
sounds more reasonable